nexus_transporter/src/TransporterNode.cpp

```diff
@@ -399,7 +399,14 @@ auto TransporterNode::on_configure(const State& /*previous_state*/)
               geometry_msgs::msg::TransformStamped tf_msg;
               tf_msg.header.stamp =
               node ? node->get_clock()->now() : rclcpp::Clock().now();
-              tf_msg.header.frame_id = "world";
+              if (state.location.header.frame_id.empty())
+              {
+                tf_msg.header.frame_id = "world";
+              }
+              else
+              {
+                tf_msg.header.frame_id = state.location.header.frame_id;
+              }
```

Comment:
Nit: Can rely on ternary expression for assignment.

Reply:
https://github.com/osrf/nexus/pull/125/commits/08fa8bfab98f68b63753d663d3bd77e5f813d700